modules/light-clients/08-wasm-light-clients/contracts/ethereum-light-client/src/contract.rs

```diff
@@ -0,0 +1,299 @@
+use cosmwasm_std::entry_point;
```

Comment:
Do we need to include cosmwasm module to use this?
Or we just use the wasm runtime to execute the code, but not require the module?

Reply:
There is a hard dependency right now on ibc wasm light clients with cosmwasm (through the 08-wasm go proxy module).